### PR TITLE
Installations: Update Chrome instructions for WSL

### PR DESCRIPTION
**Pull request by xandora**
## Because

The instructions for installing Chrome as a WSL user were not very clear, and seemed to have been copied from the macOS instructions and referenced some macOS only features.

## This PR

Provides a short description about installing Chrome and why it must be installed on Windows instead of in WSL.

## Issue

N/A

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/installations/installations.md [hunk 1]
```diff
@@ -502,6 +502,8 @@ Chrome is going to use this terminal to output various messages and won't let yo
 
 <summary class="dropDown-header">WSL2</summary>
 
+WSL does not have a graphical user inteface (GUI) like Windows, so this step will guide you through installing Google Chrome for Windows instead. Later on, we'll cover how you can use the Windows installed Chrome to preview your work that lives within WSL.
```
Comment (xandora) [inline foundations/installations/installations.md:505]:
```suggestion
WSL does not have a graphical user interface (GUI) like Windows, so this step will guide you through installing Google Chrome for Windows instead. Later on, we'll cover how you can use the Windows installed Chrome to preview your work that lives within WSL.
```

**Comment by CouchofTomato** [comment]
@xandora is this waiting for a review?

**Comment by xandora** [comment]
> @xandora is this waiting for a review?

Yea, but I just noticed that I was a dingus and PRd from my main, so other changes I had made have piggy backed along...

**Comment by CouchofTomato** [comment]
Ok. Let me know when you've got it to a place you're happy for someone to take a look.

**Comment by xandora** [comment]
> Ok. Let me know when you've got it to a place you're happy for someone to take a look.

Should be good to review. Both changes were intended to be PRd, buddy not together. 😅

**Review by CouchofTomato — APPROVED** [review]
Thanks.

**Pull request merged**